app/src/main/java/com/hedvig/app/feature/embark/passages/externalinsurer/retrieveprice/RetrievePriceActivity.kt

```diff
@@ -125,11 +124,13 @@ fun RetrievePriceScreen(
                 RetrievePriceContent(
                     onRetrievePriceInfo = viewModel::onRetrievePriceInfo,
                     onIdentityInput = viewModel::onIdentityInput,
+                    onDismissError = viewModel::onDismissError,
                     input = viewState.input,
                     title = viewState.market?.titleRes()?.let { stringResource(it) } ?: "",
                     placeholder = viewState.market?.placeHolderRes()?.let { stringResource(it) } ?: "",
                     label = viewState.market?.labelRes()?.let { stringResource(it) } ?: "",
                     inputErrorMessage = viewState.inputError?.errorTextKey?.let { stringResource(it) },
+                    errorMessage = viewState.inputError?.errorTextKey?.let { stringResource(it) }
```

Comment:
I think the error state makes sense to embed in the content composable here, since the error state originates from the viewmodel.